civihr_default_theme/scss/components/_container.scss

```diff
@@ -0,0 +1,78 @@
+/**
+* container for report
+**/
+
+.chr-separator {
+  background-color: $light-gray;
+  height: 2px;
+}
+
+.chr-container {
+  background-color: $gray-light;
+  margin-top: 60px;
```

Comment:
Actually, on a further review, what you really are trying to do is to set a `60px` top margin on the `.tab-content` element

```html
<div class="my-leave-page">
  <div class="button-container"></div>
  <div class="tabs-container">
    <div class="content">
      <!-- This is the element that should have the top margin -->
      <div class="tab-content"></div>
    </div>
  </div>
</div>
```
because you want the tab content to always have that margin, no matter if the active tab is Report or Calendar